app/seqexec-server-gn/src/main/resources/app.conf

```diff
@@ -39,6 +39,7 @@ seqexec-engine {
     odbQueuePollingInterval = "3 seconds"
     tops = "tcs=tcs:, ao=ao:, gm=gm:, gc=gc:, gws=ws:, m2=m2:, oiwfs=oiwfs:, ag=ag:, f2=f2:"
     epics_ca_addr_list = "10.2.2.255 10.2.10.21 10.2.126.101"
+    ioTimeout = "5 seconds"
```

Comment:
Where the 5 second come from? Or is it just to start testing?

Reply:
It's just to start testing.